packages/backend/jobs-manager/service/src/modules/database/container/container.model.ts

```diff
@@ -0,0 +1,17 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, Types } from 'mongoose';
+
+export type ContainerDocument = Container & Document;
+
+export class ContainerSummary {
+  id: Types.ObjectId;
+  image: string;
+}
+
+@Schema()
+export class Container {
+  @Prop({ unique: true })
+  public image!: string;
+}
+
```

Comment:
Est-ce que c'est utile d'avoir une DB de containers? Faut voir où on voit cette feature aller dans le futur, mais si l'objectif est de permettre aux utilisateurs d'utiliser n'importe-quel container, on peut simplement garder le nom de l'image dans la job. La route peut donner des suggestions de conteneurs, mais ça peut être in-memory à partir des configs.

Reply:
J'avais en tête comme objectif de permettre de construire tes propres containers à partir de l'interface, donc de pouvoir gérer ton `Dockerfile` comme les subs/jobs, avec ton nom d'image and stuff, c'est pour ça que je suis parti vers un truc en BD. 

Peut-être effectivement que le plus simple c'est de permettre d'avoir n'importe quelle image, mais soudainement si tu veux des trucs plus custom tu dois te mettre a hoster tes images, idk.

Je vois des plus aux deux use cases tbh.